src/Neo.Compiler.MSIL/FuncExport.cs

```diff
@@ -1,23 +1,25 @@
-using Neo.Compiler;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 
-namespace vmtool
+namespace Neo.Compiler
 {
     public class FuncExport
     {
-        static string ConvType(string _type)
+        internal static string ConvType(string type)
```

Comment:
```suggestion
        internal static string ConvType(Type type)
```

Can we use `Type` instead of `string`?

Reply:
Done